tools/BUILD

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+package(default_visibility = ["//visibility:public"])
+
+load(":kubectl_toolchain.bzl", "kubectl_toolchain")
+
+# kubectl toolchain type
+toolchain_type(name = "toolchain_type_kubectl")
```

Comment:
Rename this to "toolchain_type" instead of "toolchain_type_kubectl". See https://github.com/bazelbuild/rules_docker/pull/559

The recommended flow is to use the label of the package to figure out the specific type of the toolchain. Recommend creating a "//toolchains/kubectl" directory and this entire file and kubectl_toolchain.bzl there. See my PR for reference